### PR TITLE
Update dialog_thievesguild.cpp

### PR DESCRIPTION
**Pull request by 4D17Y4**
Fixed text in bar and Columns in Adventure Map 
#1590 #1593

### CODE REVIEW

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @Aditya-Srivastav, could you please take a look into given comment?

File: src/fheroes2/dialog/dialog_thievesguild.cpp [hunk 1]
```diff
@@ -283,7 +283,7 @@ void Dialog::ThievesGuild( bool oracle )
     fheroes2::Blit( fheroes2::AGG::GetICN( ICN::WELLXTRA, 2 ), display, dst_pt.x, dst_pt.y );
 
     // text bar
-    text.Set( _( "Thieves' Guild: Player RanKings" ), Font::BIG );
+    text.Set( _( "Shrine ~ Player RanKings" ), Font::BIG );
```
[the remark this answers is not in the document]
Comment (Aditya-Srivastav) [reply src/fheroes2/dialog/dialog_thievesguild.cpp:286]: As I am unfamiliar with the game.
I was able to run the game but could not reach the window in the issue.
Therefore I could not test the #1593.
I will fix #1590 for now.

[the remark this answers is not in the document]
Comment (ihhub) [reply src/fheroes2/dialog/dialog_thievesguild.cpp:286]: You can create separate pull request for #1593. It's good if we have one pull request per issue so it's easy to track changes :)

**Comment by ihhub** [comment]
Hi @4D17Y4 could you please tell us whether you're going to complete this pull request?

**Comment by ihhub** [comment]
Closing this pull request as it needs a rework and the author is not going to work on it :(

**Pull request closed without merging**